MaterialLibrary/src/main/java/io/doist/material/res/MaterialResources.java

```diff
@@ -109,9 +103,15 @@ public Drawable getDrawable(int id) throws Resources.NotFoundException {
         return res;
     }
 
-    Drawable loadDrawable(Context context, Resources resources, MaterialConfiguration configuration, TypedValue value,
-                          int id)
+    Drawable loadDrawable(Context context, Resources resources, TypedValue value, int id)
             throws Resources.NotFoundException {
+        final MaterialConfiguration configuration = mConfiguration.get(resources);
+        final int configChanges = configuration.updateConfiguration(resources);
+        if (configChanges != 0) {
+            clearDrawableCacheLocked(mDrawableCache, configChanges);
+            clearDrawableCacheLocked(mColorDrawableCache, configChanges);
+        }
```

Comment:
That's just because of `MaterialTypedArray`, as it couldn't go through `getDrawable` and I wanted to keep `MaterialConfiguration` internal. :)